refactor.py

```
@@ -1,26 +1,27 @@
 import os
+import subprocess
 
 
 def install_refactor_tool():
-    os.system("pip install ruff==0.6.0")
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', 'ruff==0.6.0'])
```

Ruff (0.8.2)

Undefined name `sys`

(F821)